addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -211,22 +208,20 @@ private void updateFanartUrl(String imagePath) {
         /*
```

Comment:
I think we should ask @pail23 here. I can remove it, but I cannot answer why it was disabled.